.eslintrc.json

```diff
@@ -66,8 +66,7 @@
         ]          
       }
     ],
-    "import/no-commonjs": 2,
```

Comment:
I don't think we need to touch the lint config here?

Reply:
oh yea, this was temporary change for development purposes lol